### PR TITLE
Prepare for release 1.24

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

**Comment by gberaudo** [comment]
@ahocevar there are CORS issues with the Boundless WMS layer used in some examples. Could you please see if the server can be fixed? Thanks a lot.

```
epsg-4326.html:1 Access to Image at 'http://demo.boundlessgeo.com/geoserver/wms?SERVICE=WMS&VERSION=1.3.0&REQUES…W_DR&WIDTH=256&HEIGHT=256&CRS=EPSG%3A4326&STYLES=&BBOX=-90%2C-180%2C90%2C0' from origin 'http://localhost:4000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://localhost:4000' is therefore not allowed access. The response had HTTP status code 503.
```

**Comment by ahocevar** [comment]
@gberaudo I have made our admins aware of the issue. I cannot promise when it will be fixed though.

**Comment by gberaudo** [comment]
Great, thanks @ahocevar.

File: package.json [hunk 1]
```diff
@@ -12,16 +12,16 @@
...
   "dependencies": {
-    "async": "2.1.2",
+    "async": "2.1.4",
     "htmlparser2": "3.9.2"
   },
   "devDependencies": {
-    "closure-util": "1.15.1",
-    "geojsonhint": "1.2.1",
-    "fs-extra": "1.0.0",
+    "closure-util": "1.17.0",
```
Comment (fredj) [inline package.json]: please use 1.18.0

Comment (fredj) [inline package.json]: and use `isolation_mode` instead of `output_wrapper` in `build/*.json`

**Comment by gberaudo** [comment]
@fredj, done and reworked the examples.

**Pull request merged**